src/lsst/cm/tools/core/error_code_decisions.yaml

```diff
@@ -241,6 +305,17 @@ pandaErrorCode:
             rescue: 0
             panda: 0
             intensity: 0.001
+        p_remote_io:
+            diagMessage: >
+                sqlalchemy.exc.InternalError: (psycopg2.errors.InternalError_) could not read block .* in file ".*":
+                Remote I/O error
+            function: mergeExecutionButler
+            known: 1
+            ticket: ["DM-37570"]
+            resolved: 1 # double check this
+            rescue: 0
```

Comment:
Merge job should be tried again after failures.  Though it's not a rescue workflow.